zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/Booter.java

```diff
@@ -71,16 +73,47 @@ static String resolveLocalRepoPath(String localRepoPath) {
     return Paths.get(home).resolve(localRepoPath).toAbsolutePath().toString();
   }
 
+  /**
+   * Check for environment variable and java property with precedence of the environment variable
+   * @param environmentVariableName
+   * @param javaPropertyName
+   * @return
+   * The environment variable, if not set the system property, or null
+   */
+  private static String getEnvironmentVariable(
```

Comment:
perhaps a different name? this isn't just getting from environment

Reply:
You are right. There is similar functionality in ZepperlinConfiguration https://github.com/apache/zeppelin/blob/5c94d7550aa569361d5c5569004917ee9bd82c70/zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java#L187 but only availabe on the config. Would something like "getEnviromentVariableOrProperty()" be okay?

Reply:
sure